cmd/create.go

```diff
@@ -193,11 +193,23 @@ func renderError(err error, ctx config.YamlCtx) string {
 }
 
 func renderRichError(err error, pos config.Pos, ctx config.YamlCtx) string {
+	line := pos.Line - 1
+	if line < 0 {
+		line = 0
+	}
+	if line >= len(ctx.Lines) {
+		line = len(ctx.Lines) - 1
+	}
+
 	pref := fmt.Sprintf("%d: ", pos.Line)
-	arrow := strings.Repeat(" ", len(pref)+pos.Column-1) + "^"
+	arrow := " "
```

Comment:
Why a single space?

Reply:
There was a strange behavior I didn't dig into much - with  empty `arrow` either format or print did swallow the last "empty" line, not sure why, but ` ` did fix it.